### ISSUE TITLE
Compare changes in old model and the new one and find out why the model does not reach the desirable performance anymore

### ISSUE THREAD

**Issue by dbeinhauer** [dbeinhauer/mcs-source#64]
Currently, we have problems training the model to the cc_norm above 0.8 (ideally to 0.9). The problem starts with using separate inh/exc layers, weights constraints, gradient clipping, adaptive learning rate and weights constraints. The problem should be probably related to one (or combination) of those. We would expect that after weights constraint the performance should drop to 0.85 (but not to 0.6) where it is now.

**Comment by dbeinhauer** [comment]
As it seems, the training of the joint model is less stable in comparison to separate model, and overall, usage of strict weight constraints does not help the stability of the training. Possibly, usage of gradient clipping might also have negative influence in the model performance. 

Now it seems that ideal learning rate is still around `0.00001`, gradient clipping does not work, adaptive learning rate for layers does not help (at least with random weights initialization), usage of hidden time step does not work now. Thankfully, the model performance using strict constraints reached approximately 0.9 cc_norm value.
